docs/geedocs/5.2.1/answer/6034724.html

```diff
@@ -161,6 +161,9 @@ <h3>To publish a database using the command line: </h3>
 <li>Publish the database to GEE Server using <code>geserveradmin --publishdb db_name --targetpath target_path</code>. For example:
 <p><code>geserveradmin --publishdb /gevol/assets/Databases/SFMapDatabase.kmmdatabase/mapdb.kda/ver001/mapdb/ --targetpath http://myserver.org</code></p>
 </li>
+<li>Publish the database to GEE Server, specifying a default globe for GEEC, using <code>geserveradmin --publishdb db_name --targetpath target_path --setcedefault</code>. For example:
```

Comment:
may be 'specifying it as default globe...'

Reply:
'specifying it as the default globe...'